packages/scvi-tools/meta.yaml

```diff
@@ -0,0 +1,27 @@
+name: scvi-tools
+description: |
+  scvi-tools is a library for developing and deploying machine learning models based on
+  PyTorch and AnnData. With an emphasis on probabilistic models, scvi-tools streamlines
+  the development process via training, data management, and user interface abstractions.
+  scvi-tools also contains easy-to-use implementations of more than 14 state-of-the-art
+  probabilistic models in the field.
+project_home: https://github.com/scverse/scvi-tools
+documentation_home: https://docs.scvi-tools.org/en/stable/
+tutorials_home: https://docs.scvi-tools.org/en/stable/tutorials/index.html
+publications:
+  - 10.1038/s41587-021-01206-w
```

Comment:
@ori-kron-wis @canergen you have a ton of publications listed on the scvi-tools website: https://scvi-tools.org/press#publications

Which would you think are relevant here? Note that the list here will be used to count citations for the scverse stats. So it doesn't harm to err on the side of including more, but they should still be relevant to the software.